src/status_im2/subs/contact.cljs

```diff
@@ -84,6 +84,26 @@
        sort
        vals)))
 
+(defn distinct-by [f coll]
+  (let [groups (group-by f coll)]
+    (map #(first (groups %)) (distinct (map f coll)))))
+
+(re-frame/reg-sub
+ :contacts/add-members-sections
```

Comment:
@OmarBasem, could you add some tests to this subscription? This is considered a layer-3 sub, we should test them, especially a non-trivial like this one. I've recently merged a [PR](https://github.com/status-im/status-mobile/pull/14472) showing how to do it, it's almost as simple as testing any other pure function.

Test examples for layer-3 subs:

- https://github.com/status-im/status-mobile/blob/develop/src/status_im2/subs/communities_test.cljs
- https://github.com/status-im/status-mobile/blob/develop/src/status_im2/subs/wallet/wallet_test.cljs
- https://github.com/status-im/status-mobile/blob/develop/src/status_im2/subs/activity_center_test.cljs

Reply:
Sure @ilmotta, thank you!

Reply:
@ilmotta I am currently unable to make the test pass successfully. I do not want to spend more time on this right now, so I have opened an issue for it (https://github.com/status-im/status-mobile/issues/14531) and will get back to it later (if you don't mind).

Reply:
It's challenging I know, not the simplest of the tests to start, given its stateful nature. Thanks for trying, perhaps let's pair program on that new issue next week.

Reply:
Thanks @ilmotta, we can try again with it sometime next week.